tests/golden/find-command/find-command.bats

```diff
@@ -131,6 +131,20 @@ EOF
     a - [2000-01-01 01:01:01]
       sortfield: c [2000-01-01 01:01:01]
 EOF
+
+  run cleanOutput coffer find / --sort name:contents:desc
+
+  assert_success
+  assert_output - <<EOF
+/
+  secrets/
+    a - [2000-01-01 01:01:01]
+      sortfield: c [2000-01-01 01:01:01]
+    b - [2000-01-01 01:01:01]
+      sortfield: b [2000-01-01 01:01:01]
+    c - [2000-01-01 01:01:01]
+      sortfield: a [2000-01-01 01:01:01]
+EOF
```

Comment:
Nice :+1: 

Since this is testing a separate edge case, let's put this assertion in its own dedicated test, e.g. `test "sort on field with name 'date' and 'name'"`

And also add a similar test for a field named `date`